sections/changes.include

```diff
@@ -5,6 +5,29 @@
 
+  <h3 id="changes-wd5">Changes between <a href="https://www.w3.org/TR/html52/">Working Draft 6</a> and <a href="https://www.w3.org/TR/2017/WD-html52-20170228/">Working Draft 5</a></h3>
+  
+  <dl>
+  <dt><a href="https://github.com/w3c/html/commit/51b402ad12a7168cb2dc47dbb3b63d6856c9384b">Fix summary activation algorithm</a></dt>
+  <dd>Removes the step that aborts activation if the <{summary}> element is not the first child of the <{details}> element, and clarifies that only the first <{summary}> element inside a <{details}> element is an interactive element.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/e205f9d142ea0870ddf8bccc2d8a31f9b5abe3df">Guidance for UI filtering for suggestions</a></dt>
+  <dd>Recommends substring matching on the <{option/label}> attribute of the <{option}> element.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/41542670a9b15ef41c074714d30827e8565b0831">Update button element definition</a></dt>
+  <dd>Improves clarity and readability of the <{button}> element definition.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/a208c83eda297b7a79cff5286e63e28cd1ffb7c3">Updating summary element rendering</a></dt>
+  <dd>Updates recommended CSS rendering for the <{summary}> element, and removes rendering expectations for a <{summary}> element that is not a child of the <{details}> element.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/e19ee96316b3dc6aa4afea4197c72eda59a3474">Make style in body conforming</a></dt>
+  <dd>Adds a note to remove the restriction for the <{style}> element being in the <{head}> element (with a warning that using the <{style}> element  in the <{body}> element  can have unexpected consequences); and adds the <{style}> element to the set of flow content elements for the <{body}> element.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/c5b7996e390e29bee7101a33604fcba5c3d4dcc0">Clarified restrictions on children of p elements</a></dt>
+  <dd>Clarifies that child elements of the <{p}> element cannot be inline blocks, inline tables, or floated and positioned block-level elements.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/c1cbc95ed3d911b23e3607a1803cfd181fbbc802">Make HTML4/xHTML1 strict doctypes non-conforming</a></dt>
+  <dd>Disallows the use of the HTML4 and xHTML 1.1 strict doctypes.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/b75ed699d0db74790ce9d61bad711cc9ecd5e053">Clarify where to propose rel extensions</a></dt>
+  <dd>References the correct part of the Microformats wiki, and adds a note to indicate proposals can also be filed against this HTML specification.</dd>
+  <dt><a href="https://github.com/w3c/html/commit/49073d89a7829dc3dbbfc3802f38bf9845902117">Remove references to Polyglot Markup</a></dt>
+  <dd>Removes references to the Polyglot Markup specification (now published as a W3C Note).</dd>
+  </dl>
```

Comment:
Missing [allow role="none" on `img`](https://github.com/w3c/html/commit/0b959a586e54fe4145b69303ebb11755cea26c09)

For `img alt=""` only